userspace/libsinsp/events/sinsp_events.h

```diff
@@ -198,6 +198,21 @@ set<ppm_event_code> sc_set_to_event_set(const set<ppm_sc_code> &ppm_sc_of_intere
 */
 set<ppm_event_code> all_event_set();
 
+/*!
+  \brief Get the ppm_event for each non sc events.
+*/
+set<ppm_event_code> all_non_sc_event_set();
```

Comment:
`all_events().filter([&](ppm_sc_code e) { return !libsinsp::events::is_syscall_event(e); });`

Isn't this equivalent? Is it worth to add and maintain API methods if we already have a filtering functionality? This applies to the ones above too. WDYT?

Reply:
Incorporated :)